package.json

```diff
@@ -6,8 +6,13 @@
   "directories": {
     "test": "tests"
   },
+  "nyc": {
+    "require": [
+      "@babel/register"
+    ]
+  },
   "scripts": {
-    "test": "npm run dbtest && nyc mocha --timeout 1000000 --exit --require @babel/register --require @babel/polyfill ./server/tests/**/*.js || true",
+    "test": "npm run dbtest && nyc mocha --timeout 1000000 --exit ./server/tests/**/*.js || true",
```

Comment:
@LukasChiama Do you the reason why the code owner put this part `|| code` in the `test` script? I know you didn't write it. I just want to know if you understand it. Thanks

Reply:
I don't know why. I thought about and planned to remove it in order to find out what it does but it slipped my memory. Thanks for raising this

Reply:
You are welcome. It's alright